operator/v1/0000_80_csi_snapshot_operator_01_crd.yaml

```diff
@@ -0,0 +1,140 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: csisnapshots.operator.openshift.io
+spec:
+  group: operator.openshift.io
+  names:
+    kind: CSISnapshot
```

Comment:
Is it too late to call this `CSISnapshotController` ?

Reply:
I'm not opposed, but would want insight from @jsafrane as well. If needed I can change this.

Reply:
CSISnapshotController sounds good to me (folowing KubeControllerManager CRD name).

Reply:
This has been renamed. The references in both `types_csi_controller.go` and this file have been updated to match.